src/line-graph.js

```diff
@@ -53,98 +31,143 @@ export class LineGraph extends HTMLElement {
       return {
         group,
         datapoints: data.filter(d => d[grouping] === group).map((d) => {
-          return { time: new Date(d.time), value: +d.mean }
+          return {
+            time: new Date(d.time),
+            value: +d.mean
+          }
         })
       }
     });
   }
+  connectedCallback() {
+    this.innerHTML = `<div style="height: 400px,width:400px" id="container"></div>`;
+    var container = document.querySelector(`#container`);
+    /*if (this.dataset.lineColor.indexOf(',') > -1) {
```

Comment:
we can remove it if it is not related to future work

Reply:
yup not needed .. removed !